policyengine_uk/variables/input/income.py

```diff
@@ -37,7 +37,9 @@ class pension_income(Variable):
     unit = GBP
     reference = "Income Tax (Earnings and Pensions) Act 2003 s. 1(1)(b)"
     quantity_type = FLOW
-    uprating = "calibration.programs.pension_income.budgetary_impact.ENGLAND"
+    uprating = (
+        "calibration.programs.private_pension_income.budgetary_impact.ENGLAND"
+    )
```

Comment:
Final thing before merging this- could we add a deprecation rather than a hard break here?

```
class pension_income(Variable):
  ... existing old code

class private_pension_income(Variable):
  ...
  def formula(...):
    # Don't have data, use the old pension income
    pension_income = person("pension_income", period)
    if pension_income.sum() != 0:
      logging.warn("`pension_income` is deprecated and will be removed- use `private_pension_income` instead")
      return pension_income
    else:
      return 0
```

Reply:
Sure, will do, thanks for the detailed explanation.